geolocator_web/example/lib/main.dart

```diff
@@ -0,0 +1,192 @@
+    githubURL: 'https://github.com/Baseflow/flutter-geolocator',
+    pubDevURL: 'https://pub.dev/packages/geolocator',
+    pages: [GeolocatorWidget.createPage()],
+  ));
+}
+
+/// Example [Widget] showing the functionalities of the geolocator plugin
+class GeolocatorWidget extends StatefulWidget {
+  /// Utility method to create a page with the Baseflow templating.
+  static ExamplePage createPage() {
+    return ExamplePage(Icons.location_on, (context) => GeolocatorWidget());
+  }
+
+  @override
+  _GeolocatorWidgetState createState() => _GeolocatorWidgetState();
+}
+
+class _GeolocatorWidgetState extends State<GeolocatorWidget> {
+  final List<_PositionItem> _positionItems = <_PositionItem>[];
+  StreamSubscription<Position>? _positionStreamSubscription;
+
+  @override
+  Widget build(BuildContext context) {
+    const sizedBox = SizedBox(
+      height: 10,
+    );
+
+    return Scaffold(
+      backgroundColor: Theme.of(context).backgroundColor,
+      body: ListView.builder(
+        itemCount: _positionItems.length,
+        itemBuilder: (context, index) {
+          final positionItem = _positionItems[index];
+
+          if (positionItem.type == _PositionItemType.permission) {
+            return ListTile(
+              title: Text(positionItem.displayValue,
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                    color: Colors.white,
+                    fontWeight: FontWeight.bold,
+                  )),
+            );
+          } else {
+            return Card(
+              child: ListTile(
+                tileColor: themeMaterialColor,
+                title: Text(
+                  positionItem.displayValue,
+                  style: TextStyle(color: Colors.white),
+                ),
+              ),
+            );
+          }
+        },
+      ),
+      floatingActionButton: Column(
+        crossAxisAlignment: CrossAxisAlignment.end,
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          FloatingActionButton.extended(
+            onPressed: _toggleListening,
+            label: Text(() {
+              if (_positionStreamSubscription == null) {
+                return "Start position updates stream";
+              } else {
+                final buttonText =
+                    _positionStreamSubscription!.isPaused ? "Resume" : "Pause";
+
+                return "$buttonText position updates stream";
+              }
+            }()),
+            backgroundColor: _determineButtonColor(),
+          ),
+          sizedBox,
+          FloatingActionButton.extended(
+            onPressed: () async {
+              await GeolocatorPlatform.instance.getCurrentPosition().then(
+                  (value) => {
+                        _positionItems.add(_PositionItem(
+                            _PositionItemType.position, value.toString()))
+                      });
+
+              setState(
+                () {},
+              );
```

Comment:
I would set this on 1 line.
```suggestion
              setState(() {});
```

Reply:
Fixed.